### PR TITLE
GEA-feature/RenovationSchedules

### PR DESCRIPTION
**Pull request by Ana00000**

### CODE REVIEW

File: GraphicEditor/View/UserControls/BuildingUserControl.xaml.cs [hunk 1]
```diff
@@ -50,7 +51,8 @@ private void SchedulesClick(object sender, RoutedEventArgs e)
                 return;
             }
             SchedulesWindow schedulesWindow = new SchedulesWindow(appointmentController.GetAll(),
-                                                            equipmentRelocationController.GetAll());
+                                                            equipmentRelocationController.GetAll(),
+                                                            renovationController.GetAll());
```
Comment (milicasiriski) [inline GraphicEditor/View/UserControls/BuildingUserControl.xaml.cs:55]: Formatting here seems suspicious 😄 might just be because of github though

**Review by milicasiriski — APPROVED** [review]
Seems good, nice PR!

**Review by lemara98 — APPROVED** [review]

**Comment by Ana00000** [comment]
[(3) Cancel scheduled actions [3]](https://trello.com/c/HvqQpuvl/102-3-cancel-scheduled-actions-3)

**Comment by Ana00000** [comment]
[(3) Show schedule for each room [3]](https://trello.com/c/KrYXfTpK/101-3-show-schedule-for-each-room-3)

**Pull request merged**